site/docs/v1/tech/installation-guide/monitor.adoc

```diff
@@ -26,9 +26,47 @@ There are a few different APIs exposing FusionAuth application specific informat
 
 In general these are APIs you will have poll to ingest.
 
+=== Log File Formats
+
+.Log File Information
+[cols="1,3,3,3,1"]
+|===
+| Log Type | Export Format | Timezone | Date Format | API Link
+
+| Audit Logs
+| Zipped CSV
+| Specified by the `zoneId` parameter
+| Specified by the `dateTimeSecondsFormat` parameter
+| link:/docs/v1/tech/apis/audit-logs/[API]
+
+| Event Logs
+| JSON
+| UTC
+| link:/docs/v1/tech/reference/data-types/#instants[Instant]
+| link:/docs/v1/tech/apis/event-logs/[API]
+
+| Login Records
+| Zipped CSV
+| Specified by the `zoneId` parameter
+| Specified by the `dateTimeSecondsFormat` parameter
+| link:/docs/v1/tech/apis/login/#export-login-records[API]
+
+| System Logs
+| Zipped directory containing log files. Lines separated by newlines. Can be unstructured (stack traces).
+| The timezone of the server
```

Comment:
This does take a `zoneId` like the others, but I think we only use it to generate the timestamp for the filename.  But otherwise, you're correct, the logs will use system time.